app/main.py

```diff
@@ -1,8 +1,41 @@
 class Car:
-    # write your code here
-    pass
+    def __init__(self, comfort_class: int, clean_mark: int, brand: str):
+        self.comfort_class = comfort_class
+        self.clean_mark = clean_mark
+        self.brand = brand
 
 
 class CarWashStation:
-    # write your code here
-    pass
+    def __init__(self,
+                 distance_from_city_center: float,
+                 clean_power: int,
+                 average_rating: float,
+                 count_of_ratings: int
+                 ):
+        self.distance_from_city_center = distance_from_city_center
+        self.clean_power = clean_power
+        self.average_rating = average_rating
+        self.count_of_ratings = count_of_ratings
+
+    def serve_cars(self, cars: list):
+        count_clear_car = 0
+        for car in cars:
+            if car.clean_mark < self.clean_power:
+                count_clear_car += self.calculate_washing_price(car)
+                self.wash_single_car(car)
+        return round(count_clear_car, 1)
+
+    def calculate_washing_price(self, car: Car):
+        diff_of_power = self.clean_power - car.clean_mark
+        calculate_mult = car.comfort_class * diff_of_power
+        calculate_div = self.average_rating / self.distance_from_city_center
+        return round(calculate_mult * calculate_div, 1)
+
+    def wash_single_car(self, car):
```

Comment:
if you started to use a type annotation in other functions then use it for every function